src/App.tsx

```diff
@@ -1,7 +1,83 @@
-import { greet } from "./utils/greet";
-
+import episodes from "./episodes.json";
+import EpisodesDisplay from "./Components/EpisodesDisplay";
+import { useEffect, useState } from "react";
+import { isMatching } from "./utils/matchingEpisodes";
+import { IEpisode } from "./utils/interface";
 function App(): JSX.Element {
-  return <h1>{greet("World")}</h1>;
+  const [searchTerm, setSearchTerm] = useState("");
+  console.log("inside the App function");
+  const [episodesData, setEpisodesData] = useState<IEpisode[]>([]);
+  useEffect(() => {
+    const getAndStoreDataFromApi = async () => {
+      const response = await fetch(
+        `https://api.tvmaze.com/shows/${episodes[0].id}/episodes`
```

Comment:
Is this "episodes" array actually carrying shows? 
Also, what if episodes[0] is undefined because the array is empty?  (perhaps that will never happen?)